collections/nemo_asr/nemo_asr/parts/asr_manifest.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2019 NVIDIA Corporation
+import json
+from typing import Union, Iterator, Dict, Any, List
+
+
+def item_iter(
+    manifests_files: Union[str, List[str]]
+) -> Iterator[Dict[str, Any]]:
+    """Iterate through json lines of provided manifests.
```

Comment:
Can you add a comment saying what fields the manifests must have?

Reply:
Added.